Fixes P5-67: Hardware test (#82)
aau-hcl-p5/main: report/4.Design/HardwareTest.tex

\section{Hardware testing}

> To ensure the best possible results, it is important to ensure the quality and precision of the chosen hardware to be at it's highest.

Reviewer: Hvad definere "best possible"
Ungå at bruge "it's"

> Therefore, a hardware test has been performed, over the motors, camera and laser, checking their precision, and quality.

Reviewer: Men vi har jo ikke kontrolleret kamera og laser.

\subsection*{Motors}

> To ensure motor capabilities, the testing has to try out combinations of precision, and to what extend the precision is reached.

Reviewer: combinations of precision?

Therefore a setup combining an angle, and a velocity ranging 0-100 is setup as seen in appendix \ref{appendix:motortest}.

The terms: Stable, Small Correction and Huge Correction is used, to specify the stopping at the angle, and to elaborate on, not only precision, but the amount of correction needed, to stop at the exact angle specified.

> The tests goal was to start at a zero degree angle, speed up to the specified velocity, and stop when reaching the target angle; Just in case the motor would go further, it would automatically speed up, in the opposite direction, until reaching its goal, hence the correction.

Reviewer: the test goal was to start... første gang du skriver i datid.

As the motors are used to adjust the target into the center of the camera-position, a certain precision at a certain speed is required.
This means that the motors used, and the movement speed should be able to stop at a certain angle, from that speed, or have a built-in correction as one of the weaknesses of the tests is, that the movement speed is either 0 or x, and not a gradient speed, which it may or may not bee in production.

The expected result and the actual result of the test are not as surprising: The higher the speed, the higher the correction.
However, it does not appear to be affected by the angle to which it has to aim, which is also observed in the testing; The zero to target speed is rather fast.

> In conclusion, in case we want to achieve the highest aim-speed as possible, we should take action upon the correction at higher speeds, and move at lower speeds when the motor approaches the target, to reduce the braking distance, thus avoiding corrections.

Reviewer: Du bør nok tilføje noget med at vi har udvalgt de motorer der har den bedste præcision i gennemsnit ved forskellige hastigheder. Altså at vi har valgt at gå med de motor der var mest stabile.

> \section{Hardware testing}

Reviewer 2: I think in general this section needs to be streamlined in terms of what tense it uses. Because I actually think that past tense is wrong.

Reviewer 2: Also I know my comments are not as good as Daniels but I am in a creative rut, and can only detect things I don't like

> To ensure the most accurate results, it is important to ensure the quality and precision of the chosen hardware to be at its highest.

Reviewer 2: Maybe change to something akin to: "to ensure the accuracy of the device"...

> To ensure the most accurate of the device, it is important to ensure the quality and precision of the chosen hardware to be at its highest.

Reviewer 3: "To ensure the most accurate of the device"

"To ensure the most accuracy of the device"

> Therefore, a test of the motors has been conducted, checking their precision and quality.

Reviewer 3: a test of the motors and the chosen camera

> Therefore, a hardware test has been performed on the motors, checking their precision and quality.

Reviewer 2: Think this should be "Therefore, a test of the motors has been conducted, ...." or something

\subsection*{Motors}

> To ensure motor capabilities, the testing has to try out combinations of target angles and movement speed, and to what extend precision is reached, considering the braking distance.

Reviewer 2: extend -> extent
also maybe just change the final part to something like: and to determine how precise the motors are. Dunno, I just dont really like that final part too much.

> To ensure motor capabilities, the testing has to try out combinations of target angles and movement speed, and to what extend precision is reached, considering the braking distance.

Reviewer 4: ```suggestion
To ensure the capabilities of the motors, the tests have to try out combinations of target angles and movement speed, and to what extend precision is reached, considering the braking distance.
```

den er stadig lidt skrald, hjælp @nillas12

> Therefore a setup combining an angle, and a velocity ranging 0-100 is setup as seen in appendix \ref{appendix:motortest}.

Reviewer 4: ```suggestion
Therefore a setup combining an angle, and a velocity ranging 0-100 is setup as seen in Appendix~\ref{appendix:motortest}.
```

> Therefore a setup combining an angle, and a velocity ranging 0-100 is setup as seen in appendix \ref{appendix:motortest}.

Author: 0 er stilstand, og også en "velocity", vel?

> Therefore a setup combining an angle, and a velocity ranging 0-100 is setup as seen in Appendix~\ref{appendix:motortest}.

Reviewer 3: "Ranging 0-100" giver ikke mening for mig... (eller jo men det lyder forkert)
"ranging from 0 to 100"
eller "in range 0 to 100" (eller 0-100)

> The terms: Stable, Small Correction and Huge Correction is used, to specify the stopping at the angle, and to elaborate on, not only precision, but the amount of correction needed, to stop at the exact angle specified.

Reviewer 4: are used

Reviewer 4: btw skal de stå med store?

Reviewer 4: ```suggestion
The terms stable, small correction and huge correction are used to specify the stopping at the angle, and to elaborate on, not only precision, but the amount of correction needed, to stop at the exact angle specified.
```

jeg forstår ikke "to specity the stopping at the angle" sætningen, er jeg retarderet???????

> The terms stable, small correction and huge correction are used to specify the stopping at the angle, and to elaborate on, not only precision, but the amount of correction needed, to stop at the exact angle specified.

Reviewer 3: hvis du bruger et term skal det på en eller anden måde have highlighting på at det er termet.. så enten italic eller citationstegn or something. vi bruger vist generelt italic i rapporten.

> The tests goal is to start at a zero degree angle, speed up to the specified velocity, and stop when reaching the target angle; Just in case the motor will go further, it will automatically speed up, in the opposite direction, until reaching its goal, hence the correction.

Reviewer 4: ```suggestion
The tests goal is to start at a zero degree angle, speed up to the specified velocity, and stop when reaching the target angle; just in case the motor will go further, it will automatically speed up, in the opposite direction, until reaching its goal, hence the correction.
```

Kun stort efter semikolon hvis det er et proper noun.

> The tests goal is to start at a zero degree angle, speed up to the specified velocity, and stop when reaching the target angle; Just in case the motor will go further, it will automatically speed up, in the opposite direction, until reaching its goal, hence the correction.

Reviewer 2: The tests? I am genuinely not sure if it should be tests, test's, tests'. But for some reason tests ring wrong to me.

Reviewer 2: Also, while this is technically ok, I do think this sentence is a bit long (thats what she said, although it was not in reference to a sentence, but rather to ma dick)

> The tests' goal is to start at a zero degree angle, speed up to the specified velocity, and stop when reaching the target angle; just in case the motor will go further, it will automatically speed up, in the opposite direction, until reaching its goal, hence the correction.

Reviewer 3: "the goal of the tests" synes jeg lyder bedre.

Reviewer 3: "speed up" er vel forkert i dine tests. det er vel "vi kører med hastighed X eller ikke med hastighed X". så "set speed" eller lign

> As the motors are used to adjust the target into the center of the camera-position, a certain precision at a certain speed is required.

Reviewer 4: vil gerne af med den ene certain

> As the motors are used to adjust the target into the center of the camera-position, a certain precision at a certain speed is required.

Reviewer 2: I don't like the final part of this sentence

> As the motors are used to adjust the target into the center of the camera-position, a certain precision at a given speed is required.

Reviewer 3: as the motors are used to aim, precision is a high valued quality.

> This means that the motors used, and the movement speed should be able to stop at a certain angle, from that speed, or have a built-in correction as one of the weaknesses of the tests is, that the movement speed is either 0 or x, and not a gradient speed, which it may or may not bee in production.

Reviewer 4: either 0 or x?

Gradient speed? Varying?

og der skal helst ikke være bier i produktionen

> This means that the motors used, and the movement speed should be able to stop at a certain angle, from that speed, or have a built-in correction as one of the weaknesses of the tests is, that the movement speed is either 0 or x, and not a varying velocity, which it may or may not be in production.

Reviewer 3: "this means that the motors used, should be able to stop at a certain angle, when running at speed x. " er bedre. men stadig en mærkelig del-sætning.

"or have a built-in correction as one of the weaknesses of the tests is, that the movement speed is either 0 or x, and not a varying velocity, which it may or may not be in production."
virker som volapyk. men split evt hele paragraffen op i flere sætninger.

> The expected result and the actual result of the test are not as surprising: The higher the speed, the higher the correction.

Reviewer 2: Would it not be "...correction required"? Or do I misunderstand

> The expected result and the actual result of the test are not as surprising: The higher the speed, the more correction required.

Reviewer 3: ret til: "the more correction *is* required"

"The expected result and the actual result of the test are not as surprising" -> betyder det at relationen mellem de 2 ikke er overraskende? eller at det endelige resultat ikke er?

> However, it does not appear to be affected by the angle to which it has to aim, which is also observed in the testing; The zero to target speed is rather fast.

Reviewer 4: ikke stort efter semikolon

> However, it does not appear to be affected by the angle to which it has to aim, which is also observed in the testing; the zero to target speed is rather fast.

Reviewer 3: "the zero to target speed is rather fast."

ret til:
"Acceleration to the target speed happens rather fast"

> In conclusion, in case we want to achieve the highest aim-speed as possible, we should take action upon the correction at higher speeds and move at lower speeds when the motor approaches the target, to reduce the braking distance, thus avoiding corrections.

Reviewer 4: undgår vi helt corrections?

Author: Hvis vi kommer langt nok ned i bremseafstand gør vi, jf. bilaget ("Stable")

> In conclusion, in case we want to achieve the highest aim-speed as possible, we should take action upon the correction at higher speeds and move at lower speeds when the motor approaches the target, to reduce the braking distance, thus avoiding corrections.

Reviewer 2: This sentence reads a bit strange. It might actually be ok, but it just gets a bit long I think

> In conclusion, in case we want to achieve the highest aim-speed as possible, we should take action upon the correction at higher speeds and move at lower speeds when the motor approaches the target, to reduce the braking distance, thus avoiding corrections.

Reviewer 3: "In conclusion, in case we want to achieve the highest aim-speed as possible" ->
"In conclusion, in case we want to achieve the highest possible aim-speed"

> The results of the tests are therefore used to not only choose which motors to use, but also giving a pointer to an optimal movement speed.

Reviewer 2: I don't like the use of "giving a pointer".

\subsection*{Camera}

> Due to the use of the camera, the actual quality of the footage is secondary.

Reviewer 3: jeg ville måske skrive noget andet og sige at det var vigtigt at vi nogenlunde klart kan se objektet på en video feed. Dette testede vi kort ved at tage nogle videoer hvor vi bevægede en bold i forskellige hastigheder, og tjekkede hvor sløret billedet blev. Den reelle opløsning er ikke vigtigt, da vi ikke skal kunne se motiver klart, men hellere fokuserer på eksekveringshastigheden. mhtFramerate er kameraet ikke en bottleneck, men nærmere vores software. Derdover bliver opløsningen nok nedskaleret.

> In this case, the camera footage will be scaled down to a lower resolution and frame rate, rendering the actual camera quality irrelevant to a certain extend.

Reviewer 2: extend -> extent

> In this case, the camera footage will be scaled down to a lower resolution and frame rate, rendering the actual camera quality irrelevant to a certain extend.

Reviewer 4: er den her sektion færdig? synes at der mangler noget dybde

Author: Det var dét eller ud med den.. Problemet er at der intet er at sige her.
